geocity/tests/submissions/test_a_permit_request.py

```diff
@@ -2631,6 +2633,29 @@ def _add_fields_select_price_and_save(self, submission):
             data=data,
         )
 
+    def test_price_at_0chf_and_submit(self):
```

Comment:
Maybe remove the currency in the function and use term "free" to be more generic.